WordPress/Classes/Services/PostAutoUploadInteractor.swift

```diff
@@ -46,7 +46,6 @@ final class PostAutoUploadInteractor {
         guard post.isFailed,
             let status = post.status,
             !PostAutoUploadInteractor.disallowedStatuses.contains(status),
-            !(post is Page),
```

Comment:
This will put `develop` in an inconsistent state. With this remotion, I'm able to tap "Publish" in a Page and the apps tell me it will upload my draft when I'm back online.

When I go back online, the page is uploaded as a draft.

I wonder if we should put this under a FeatureFlag.

Thoughts?

Besides that, everything looks ok ✅

Reply:
Actually that's unintended.  I will feature flag this check.

Good catch @leandroalonso !

Reply:
This was resolved by:

1. Adding the feature flag.
2. Retargetting this PR to the feature branch so that it won't disrupt the work of other devs.